### DISCUSSION TITLE
Is there such thing as an online zod playground?

### DISCUSSION THREAD

**Discussion by zomars** [colinhacks/zod#1303] [Ideas]
It would be nice to test and come up with schemas similarly on how we can have a TS playground online.

**Comment by scotttrinh** [comment]
TypeScript playground itself [supports importing `zod`](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbzgLzgXzgMyhEcBEyEAJvgNwBQFAxhAHYDO8AYhBHALwoB0EARgCsAptRgAKJJjYAuHkyjA6AczEBKdKqA) and I have a [CodeSandbox template](https://codesandbox.io/s/zod-template-r67dv) that I use when I actually want to run stuff.

**Reply by zomars** [reply]
Awesome! But you cannot actually test parsing on Typescript playground right? You would need to use CodeSandbox for that right?

**Reply by iambryanhaney** [reply]
TS Playground can run the code and has an output log.

<img width="1091" alt="Screen Shot 2022-08-16 at 12 58 34 PM" src="https://user-images.githubusercontent.com/61446127/184947859-a8c31701-59be-412a-992f-ca184c0701dd.png">

**Reply by iambryanhaney** [reply]
If you'd like to experiment with schema rules / parsing in rapid succession, and don't need to inspect the types, you can also just `const { z } = require('zod');` in a node REPL started in any directory that contains `node_modules/zod`.
